src/planwise/client/components/nav.cljs

```diff
@@ -3,12 +3,17 @@
 
 ;; Navigation components
 
-(defn- li-menu-item [{:keys [selected item href title icon]}]
+(defn- li-menu-item [{:keys [selected item href title icon tab-number wizard-state]}]
   (let [is-selected? (or (= selected item)
                          (item selected))]
-    [:li {:class (when is-selected? "active")}
+    [:li {:class (str
+                  (when is-selected? "active ")
+                  (when wizard-state (str (name wizard-state) " wizard")))}
```

Comment:
Why do you think the `wizard` class is redundant? It's has styles associated with it.